docs/storefront/stencil/themes/context/object-reference/schemas/settings.yml

```diff
@@ -154,6 +156,7 @@ properties:
   region_code:
     type: string
     description: The [ISO-3166-2](https://en.wikipedia.org/wiki/ISO_3166-2) code is an international standard denomination for country subdivisions. The first part is the country's ISO 3166-1 alpha-2 code, and the second part is a string of up to three alphanumeric characters representing the country's subdivision (province or state). For example, Texas's ISO 3166-2 code is US-TX; therefore, the region code is TX.
```

Comment:
Can you add something like the following: The value is approximate as it is based on the IP address geolocation. Alternatives to this deprecated field are solution like https://ipinfo.io/products/ip-geolocation-api, https://www.bigdatacloud.com/ip-geolocation and others. While those solution are not free, they offer a generous free tier and offer better quality of the data and more data points than just the region.